go-controller/pkg/cni/helper_linux.go

```diff
@@ -309,41 +264,79 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 			klog.Warningf("Failed to clear stale OVS port %q iface-id %q: %v\n  %q", uuid, ifaceID, err, out)
 		}
 	}
-
 	ipStrs := make([]string, len(ifInfo.IPs))
 	for i, ip := range ifInfo.IPs {
 		ipStrs[i] = ip.String()
 	}
-
 	// Add the new sandbox's OVS port
 	ovsArgs := []string{
-		"add-port", "br-int", hostIface.Name, "--", "set",
-		"interface", hostIface.Name,
+		"add-port", "br-int", hostIfaceName, "--", "set",
+		"interface", hostIfaceName,
 		fmt.Sprintf("external_ids:attached_mac=%s", ifInfo.MAC),
 		fmt.Sprintf("external_ids:iface-id=%s", ifaceID),
 		fmt.Sprintf("external_ids:ip_addresses=%s", strings.Join(ipStrs, ",")),
-		fmt.Sprintf("external_ids:sandbox=%s", pr.SandboxID),
+		fmt.Sprintf("external_ids:sandbox=%s", sandboxID),
 	}
 
 	if out, err := ovsExec(ovsArgs...); err != nil {
-		return nil, fmt.Errorf("failure in plugging pod interface: %v\n  %q", err, out)
+		return fmt.Errorf("failure in plugging pod interface: %v\n  %q", err, out)
 	}
 
-	if err := clearPodBandwidth(pr.SandboxID); err != nil {
-		return nil, err
+	if err := clearPodBandwidth(sandboxID); err != nil {
+		return err
 	}
 
 	if ifInfo.Ingress > 0 || ifInfo.Egress > 0 {
-		l, err := netlink.LinkByName(hostIface.Name)
+		l, err := netlink.LinkByName(hostIfaceName)
 		if err != nil {
-			return nil, fmt.Errorf("failed to find host veth interface %s: %v", hostIface.Name, err)
+			return fmt.Errorf("failed to find host veth interface %s: %v", hostIfaceName, err)
 		}
 		err = netlink.LinkSetTxQLen(l, 1000)
 		if err != nil {
-			return nil, fmt.Errorf("failed to set host veth txqlen: %v", err)
+			return fmt.Errorf("failed to set host veth txqlen: %v", err)
```

Comment:
s/veth/interface

Reply:
not directly related to this PR IMO.
(although tis is a bit missleading with SR-IOV switchdev)

Reply:
Let fix it in different PR